### PR TITLE
Improve STS related docs

### PR DESCRIPTION
**Pull request by dastergon**
After some troubleshooting around STS, I realized that some areas need an update to reflect the latest STS changes. 
I updated the configmap generation script and the docs.

Needs #574 to be merged first

PTAL

### CODE REVIEW

File: docs/1.0-Installation.md [hunk 1]
```diff
@@ -175,13 +176,13 @@ The operator also needs a ConfigMap that has:
...
         "name": "aws-account-operator-configmap",
-        "namespace": "aws-account-operator",
+        "namespace": "aws-account-operator"
     }
 }
 ```
 The ConfigMap could be generated and deployed with the `hack/scripts/set_operator_configmap.sh` script.
 
-    .hack/scripts/set_operator_configmap.sh -a ${ACCOUNT_LIMIT} -v ${VCPU_QUOTA} -r "${OSD_STAGING_1_OU_ROOT_ID}" -o "${OSD_STAGING_1_OU_BASE_ID}"
+    .hack/scripts/set_operator_configmap.sh -a ${ACCOUNT_LIMIT} -v ${VCPU_QUOTA} -r "${OSD_STAGING_1_OU_ROOT_ID}" -o "${OSD_STAGING_1_OU_BASE_ID}" -s ${AWS_ARN}
```
Comment (iamkirkbater) [inline docs/1.0-Installation.md:185]: Using the env vars that you've set in the README, wouldn't this be `-s ${STS_JUMP_ROLE_ARN}`

**Review by iamkirkbater — CHANGES_REQUESTED** [review]

**Comment by dastergon** [comment]
@iamkirkbater I've updated the PR with your suggestions! Thanks :) PTAL

**Comment by iamkirkbater** [comment]
This looks great!  Feel free to remove the hold on monday after #574 is merged and let this one merge in.

/lgtm
/hold

**Pull request merged**